### PR TITLE
copy photoswipe default-skin to build folder

### LINKED ISSUE #614: icons in the image gallery have disappeared
the top right buttons (close, full-screen) have disappeared, apparently a path error where the png file with the icons is no more found (http://c2corgv6.demo-camptocamp.com/static/f72c289/build/default-skin.png)

### PR DESCRIPTION
**Pull request by ginold**
fixes #614

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -282,6 +282,10 @@ c2corg_ui/static/build/deps.js: $(LIBS_JS_FILES) c2corg_ui/static/build/locale_m
 c2corg_ui/static/build/locale_moment: .build/node_modules.timestamp
 	cp -r node_modules/moment/locale/ c2corg_ui/static/build/locale_moment
 
+# copy skins of photoswipe.js
+photoswipe-skins: .build/node_modules.timestamp
+	cp -r node_modules/photoswipe/dist/default-skin/default-skin.png c2corg_ui/static/build/default-skin.png
```
Comment (tsauerwein) [inline Makefile]: You don't need the flag `-r` (recursive) when copying a single file.

Comment (ginold) [inline Makefile]: yep, i removed the flag and i added a call to this function under 'less'.

**Comment by ginold** [comment]
@asaunier is this PR ok?

**Comment by asaunier** [comment]
For me it's OK but it would be more relevant that @tsauerwein gives you the clearance since he has made many comments in the PR and the issue. @tsauerwein OK for you?

**Comment by tsauerwein** [comment]
:+1:

**Pull request merged**